psc/src/main/java/com/pinterest/psc/consumer/PscConsumer.java

```diff
@@ -1604,6 +1604,44 @@ public MessageId committed(TopicUriPartition topicUriPartition) throws ConsumerE
         }
     }
 
+    // TODO: unit test this
+    public Collection<MessageId> committed(Collection<TopicUriPartition> topicUriPartitions) throws ConsumerException, ConfigurationException {
```

Comment:
Please add javadoc.
Since the implementation is a bit more involved that piggybacking on the above implementation, we should also add some integration tests.